tests/foreman/ui/test_ansible.py

```diff
@@ -0,0 +1,121 @@
+"""Test class for Ansible Roles and Variables pages
+
+:Requirement: Ansible
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: Ansible
+
+:Assignee: dsynk
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from airgun.session import Session
+
+from robottelo.datafactory import gen_string
```

Comment:
Please use fauxfactory here
```suggestion
from fauxfactory import gen_string
```